scripts/publish.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
where does this script get used again?

Reply:
I know it was from an earlier PR from Dan. Is it for the workflow to publish to the local registry?

Reply:
That get used in the push-image workflow here. I added it back after it was removed by github-config.

https://github.com/jericop/jammy-tiny-stack/blob/fix-package-receipts/.github/workflows/push-image.yml#L70